packages/backend/src/api-impl.ts

```diff
@@ -103,6 +103,16 @@ export class BootcApiImpl implements BootcApi {
     return images;
   }
 
+  async inspectImage(image: ImageInfo): Promise<podmanDesktopApi.ImageInspectInfo> {
+    try {
+      return await podmanDesktopApi.containerEngine.getImageInspect(image.engineId, image.Id);
+    } catch (err) {
+      await podmanDesktopApi.window.showErrorMessage(`Error inspecting image: ${err}`);
```

Comment:
If this fails we shouldn't pop up error messages from main. The caller (likely frontend in this case) should decide the user behaviour when you can't get the info.